### PR TITLE
Add documentation for OTLP Endpoint

### LINKED ISSUE #175: Update otel to lgtm task to use grafana cloud otlp endpoints
### Request

https://grafana.com/docs/alloy/latest/tasks/opentelemetry-to-lgtm-stack/

### Use case

When this guide was written Grafana Cloud did not have otlp endpoints for Loki and Mimir. Now that we have them we should use them!

### PR DESCRIPTION
**Pull request by proffalken**
#### PR Description

Update the documentation so that it shows how to connect to the Grafana Cloud OTLP endpoint

#### Which issue(s) this PR fixes

Fixes #175 

#### PR Checklist

- [ x ] Documentation added

### CODE REVIEW

File: docs/sources/tasks/opentelemetry-to-lgtm-stack.md [hunk 1]
```diff
@@ -77,14 +77,41 @@ The pipeline currently looks like this:
 Metrics, Logs, Traces: OTLP Receiver → batch processor → OTLP Exporter
 ```
 
+## Grafana Cloud
+
+Grafana Cloud provides OTLP Endpoints that you can use directly from within {{< param "PRODUCT_NAME" >}}.
+
+You can find the OTLP connection details from the OpenTelemetry **Details** page in the [Grafana Cloud Portal][].
+
+You will need to update the configuration file as follows:
```
Comment (clayton-cornell) [inline docs/sources/tasks/opentelemetry-to-lgtm-stack.md:86]:
```suggestion
You must update the configuration file as follows:
```

**Comment by proffalken** [comment]
Both proposed changes accepted, thanks!

File: docs/sources/tasks/opentelemetry-to-lgtm-stack.md [hunk 2]
```diff
@@ -77,14 +77,46 @@ The pipeline currently looks like this:
...
+}
+```
+
+Replace the following:
+
+* _`<ACCOUNT ID>`_: Your Grafana Cloud account ID.
+* _`<API TOKEN>`_: Your Grafana Cloud API token.
+* _`<OTLP_ENDPOINT>`_: Your OTLP endpoint.
+This will use the credentials stored in `otelcol.auth.basic "default"` to authenticate against the Grafana Cloud OTLP endpoints and you should start
+to see your data arrive!
```
Comment (clayton-cornell) [inline docs/sources/tasks/opentelemetry-to-lgtm-stack.md:108]:
```suggestion

This configuration will use the credentials stored in `otelcol.auth.basic "default"` to authenticate against the Grafana Cloud OTLP endpoints, and you should start to see your data arrive!
```

One more small suggestion.

**Comment by clayton-cornell** [comment]
I'm guessing we can backport this to Alloy 1.0 and 1.1.

**Review by clayton-cornell — APPROVED** [review]
I think this is good enough. There's still some passive voice, but we could go back and forth a lot to weed it out. Better we merge and get the update published :-)

**Pull request merged**